listeners.go

```diff
@@ -468,20 +432,22 @@ func (na NetworkAddress) ListenQUIC(ctx context.Context, portOffset uint, config
 			}
 		}
 
-		sqs := newSharedQUICState(tlsConf, activeRequests)
+		sqs := newSharedQUICState(tlsConf)
 		// http3.ConfigureTLSConfig only uses this field and tls App sets this field as well
 		//nolint:gosec
 		quicTlsConfig := &tls.Config{GetConfigForClient: sqs.getConfigForClient}
-		earlyLn, err := quic.ListenEarly(h3ln, http3.ConfigureTLSConfig(quicTlsConfig), &quic.Config{
-			Allow0RTT: true,
-			RequireAddressValidation: func(clientAddr net.Addr) bool {
-				// TODO: make tunable?
-				return sqs.getActiveRequests() > 1000
-			},
-		})
+		// Require clients to verify their source address when we're handling more than 1000 handshakes per second.
+		// TODO: make tunable?
+		limiter := rate.NewLimiter(1000, 1000)
```

Comment:
This should probably be made tunable eventually, but at least we have a high limit right now that should work for most people. Maybe this could be in a follow-up PR. (It will need to be marked as experimental though as this API is still undergoing finalization.)

Reply:
In my opinion, don't fix a problem when it's not 😄 . But keep the possibility open.

Reply:
Yeah, true. If we get complaints of errors/DoS, or get requests for it, we can make this tunable I suppose.